### PR TITLE
lsappinfo: add page

### PR DESCRIPTION
**Pull request by taher-mosbah**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by vitorhcl — CHANGES_REQUESTED** [review]
Thanks for your contribution! I have some suggestions

**Comment by taher-mosbah** [comment]
Thanks for reviewing and sorry about the forced push, I signed CLA and implemented suggestions.

File: pages/osx/lsappinfo.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# lsappinfo
+
+> Control and query CoreApplicationServices about the app state on the system.
```
Comment (vitorhcl) [inline pages/osx/lsappinfo.md:3]:
```suggestion
> Control and query CoreApplicationServices about the app state on the system.
> More information: <https://keith.github.io/xcode-man-pages/lsappinfo.8.html>.
```

File: pages/osx/lsappinfo.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- List all running applications:
```
Comment (vitorhcl) [inline pages/osx/lsappinfo.md:5]:
```suggestion
- List all running applications with their details:
```

**Review by gutjuri — APPROVED** [review]
Thanks!

**Review by vitorhcl — CHANGES_REQUESTED** [review]
LGTM, after you consider my suggestions:
- It's always good to have a "More information" link when possible
- I made it clear that the `lsappinfo list` command also shows the details of the apps.

Welcome to tldr and thanks for your contribution!

**Review by vitorhcl — APPROVED** [review]
Thank you!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**